Firebase/Core/FIRObjectSwizzler.mm

```diff
@@ -0,0 +1,163 @@
+// Copyright 2017 Google
```

Comment:
Why is this and other files .mm and not .m? I don't see C++ in all of them.

Reply:
This was my mistake. I had originally written it another way

Reply:
It turns out we need these files to be .mm as they import FIRSwizzlingCaches.h which has a std::unordered_map as part of its public api.